Branch-SDK/src/io/branch/referral/ServerRequest.java

```diff
@@ -137,6 +138,11 @@ public String getRequestUrl() {
      *             as key-value pairs.
      */
     protected void setPost(JSONObject post) {
+        try {
+            post.put("metadata", prefHelper_.getRequestMetadata());
```

Comment:
Also can we rename this form `metadata` to some thing else. Already ActionCompleted Request and RegisterView Request has metadata param